src/lib/environment.ts

```diff
@@ -19,7 +21,7 @@ export function createSagaEnvironment<State>(
         throw new SagaCancelledError(`Saga has been cancelled`);
       }
 
-      store.dispatch(action);
+      return store.dispatch(action);
```

Comment:
environment's `dispatch` is marked as void, while the `store.dispatch` is `any`. Do we really want to return this?

Reply:
Probably for legacy / OMS for a while. I think it should be straightforward to implement one in terms of the other, though maybe it's not desirable to have to.

Porting existing tests should not be a big deal, so as long as we don't drop functionality, let's remove this then.